extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml

```diff
@@ -126,5 +154,25 @@
         <label>Total Activity (s)</label>
         <description>How long has the device been switched on totally</description>
     </channel-type>
+
+   <!-- Brightness Channel -->
+    <channel-type id="brightness">
+        <item-type>Dimmer</item-type>
+        <label>Brightness</label>
+        <description>The brightness channel allows to control the brightness of a light.
+            It is also possible to switch the light on and off.
+        </description>
+        <category>DimmableLight</category>
+    </channel-type>
+
+    <!-- Color Channel -->
```

Comment:
I can't see where this channel is used, so just to make sure: Is a reference to it missing anywhere?